io.github.harmonoid.yml

```diff
@@ -0,0 +1,88 @@
+app-id: io.github.harmonoid
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Platform
+rename-icon: harmonoid
+rename-appdata-file: harmonoid.appdata.xml
+rename-desktop-file: harmonoid.desktop
+command: harmonoid
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-music
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: libmediainfo
+    subdir: Project/GNU/Library
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --with-libcurl
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: https://github.com/MediaArea/MediaInfoLib/archive/refs/tags/v21.03.tar.gz
+        sha256: b083c113adb80f6608d5e86d1be07f9abd4257bd0e8456c8f9eb8756a385c5a0
+
+  - name: webkitgtk
```

Comment:
if you use the GNOME runtime this is included.

Reply:
done